<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sktinfoextractor</class>
 <widget class="QDialog" name="sktinfoextractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touch Data Extractor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>501</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Processing</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Tuio Server</string>
   </property>
   <widget class="QLineEdit" name="tuioHost">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>26</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Host : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>28</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="tuioPort">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>23</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>3333</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="sendTuioCheck">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Tuio Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Verbose (Console)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="framesPerSec">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>543</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>547</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames/sec</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="tuioMouse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Calibration</string>
   </property>
   <widget class="QGroupBox" name="twoCamsMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Two Cameras</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>26</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Avoid overlap :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>23</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="mixActive">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>24</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Mixing Active</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>117</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>129</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Point Calibration</string>
    </property>
    <widget class="QPushButton" name="pointCalStart">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Point Calibration</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Normalize Coordinates</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Num of Points :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>28</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance from border :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numPointsSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="distanceFromBorder">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>23</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <double>0.300000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.005000000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Frames for each point :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="framesPointBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Data Collector</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Touch Stabilization</string>
    </property>
    <widget class="QDoubleSpinBox" name="minDistanceToUpdateBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>25</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Threshold : </string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>248</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>370</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SKT INFO EXTRACTOR</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>26</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="numThreadsText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>543</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>546</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum threads :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
